app/services/hyrax/custom_queries/find_collections_by_type.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+module Hyrax
+  module CustomQueries
+    # @example
+    #   collection_type = Hyrax::CollectionType.find(1)
+    #
+    #   Hyrax.custom_queries.find_collections_by_type(global_id: collection_type.to_global_id)
+    #
+    class FindCollectionsByType
+      def self.queries
```

Comment:
This is unexplained, like I'm missing something.

Reply:
https://github.com/samvera/valkyrie/wiki/Queries#custom-queries

Reply:
I was expecting to see documentation. (Maybe even a Valkyrie provided builder, but this works).

Reply:
I think I'd like to see either documentation OR a spec tightening up the connection to Valkyrie.

Reply:
i added a `@see` to the above.

i think the tests are already tightly bound to Valkyrie, since this class will raise errors without a `query_service:` initializer argument.